AbstractSQLRules2SQL.ometajs

```diff
@@ -743,6 +744,23 @@ export ometa AbstractSQLRules2SQL {
 			)
 		),
 
+	Case :indent =
+		[
+			'Case'
+			[	When(indent)+:whens
+			]
+			AnyValue(indent):val
+		]
+		-> (' CASE ' + whens.join(' ') + ' ELSE ' + val + ' END'),
+
+	When :indent =
+		[
+			'When'
+			BooleanValue(indent):matches
+			AnyValue(indent):resultValue
+		]
+		-> (' WHEN ' + matches + ' THEN ' + resultValue + ' '),
```

Comment:
Do you know if we ever need brackets around the `matches` or `resultValue`?  Or is that automatically handled within them? (it might well be)

Reply:
I believe that because I set them as `BooleanValue` that the brackets would be handled as-per normal; not sure about the result value though. Do you have a scenario in mind?

Reply:
I think it's fine actually and brackets will be automatically added on the sub-parts that need them and we don't need them here